src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -1,22 +1,18 @@
 package core.mate.academy.model;
 
 public abstract class Machine implements Workable {
+    public static final String DEFAULT_NAME = "Pussy_waggon";
+    public static final String DEFAULT_COLOUR = "White";
```

Comment:
```suggestion
    private static final String DEFAULT_NAME = "Pussy_waggon";
    private static final String DEFAULT_COLOUR = "White";
```
also let's add a more appropriate value to the DEFAULT_NAME